crates/tasks/src/lib.rs

```diff
@@ -304,6 +308,25 @@ pub struct TaskExecutor {
 // === impl TaskExecutor ===
 
 impl TaskExecutor {
+    /// Returns the current `TaskExecutor` if one has been initialized
+    pub fn current() -> Option<Self> {
```

Comment:
I'd like to rename this try_current

and have current -> Self that panics if no one is set

then we need a # Panics section in the docs as well